src/components/helper/AxiosHelper.tsx

```diff
@@ -11,10 +11,7 @@ export const getAllTransactions = async () => {
   try {
     const user = getUserToken()
     if (!user) {
-      return {
-        status: 'error',
-        message: 'login first',
-      }
+      throw new Error('Token does not exist!')
```

Comment:
rather than throwing errors in 3 different functions for the same reason could this logic not be in the function that retrieves the token?